test/spec/xml/edit.js

```diff
@@ -162,20 +165,20 @@ describe('bpmn-moddle - edit', function() {
 
       // when
       // set global namespace information
-      definitions.$attrs['xmlns:di'] = 'http://www.omg.org/spec/DD/20100524/DI';
-      definitions.$attrs['xmlns:dc'] = 'http://www.omg.org/spec/DD/20100524/DC';
+      definitions.$attrs['xmlns:dodo'] = 'http://www.omg.org/spec/DD/20100524/DI';
+      definitions.$attrs['xmlns:doco'] = 'http://www.omg.org/spec/DD/20100524/DC';
```

Comment:
How is this related to the test changes?

Reply:
It does not contribute anything to the test failure. Just checked whether this special case works. I've reverted that change via https://github.com/bpmn-io/bpmn-moddle/pull/85/commits/a8a9f071a5e3718ac6dad3fb3b13f21303428c0d.